cmd.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/sh
 
-GUNICORN_WORKERS="${GUNICORN_WORKERS:-8}"
-
 ./manage.py collectstatic --noinput
 
-wait-for-it.sh "${DJANGO_DATABASE_HOST}":"${DJANGO_DATABASE_PORT}" -t "${WAITFORIT_TIMEOUT}" -- \
-    ./manage.py migrate --no-input && \
-    gunicorn --workers=$GUNICORN_WORKERS --bind=0.0.0.0:80 timed.wsgi:application
+wait-for-it.sh "${DJANGO_DATABASE_HOST}":"${DJANGO_DATABASE_PORT}" -t "${WAITFORIT_TIMEOUT}" --
```

Comment:
We already have a shell script. I think it'd be a good point in time to rewrite it in a more readable way.

You can put `set -x` on top of the script to ensure we don't continue in case of failure. (Also see our Shell coding guidelines for further info on flags etc)